packages/browser/src/util/Fetcher.ts

```diff
@@ -27,20 +27,18 @@
 /**
  * A wrapper method to wrap the standard w3 fetch library
  */
-import _fetch from "cross-fetch";
 
 export interface IFetcher {
   fetch(url: RequestInfo, init?: RequestInit): Promise<Response>;
 }
 
+// TODO: This class should be removed altogether, and replaced by a direct call
+// to the environment's fetch.
 /**
  * @hidden
  */
 export default class Fetcher implements IFetcher {
   async fetch(url: RequestInfo, init?: RequestInit): Promise<Response> {
-    if (typeof window !== "undefined" && typeof window.fetch !== "undefined") {
-      return window.fetch(url, init);
-    }
-    return _fetch(url, init);
+    return global.fetch(url, init);
```

Comment:
`global` is a Node-specific global, IIRC. Either `window` or `globalThis` (no IE support, but :shrug: ) would be more appropriate, I think: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis

Reply:
Good point, that's what I've been using everywhere else I don't know what I made an exception here. Fixed in b3fc9e4ce68dc0264b70f5ee23e959eb76f03d07